webpack/JobWizard/steps/form/SearchSelect.js

```diff
@@ -48,7 +50,7 @@ export const SearchSelect = ({
     ...selectOptions,
     ...Immutable.asMutable(response?.results || [])?.map((result, index) => (
       <SelectOption key={index + 1} value={result.id}>
-        {result.name}
```

Comment:
We should be able to see if `result` has some other properties besides `name`. I would guess it's a host object, and should have a `display_name`

Reply:
here we are using gql for the hosts query, which means that we don't have the `display_name` property.